sdk/digitaltwins/azure-digitaltwins-core/API design.md

```diff
@@ -1178,44 +1178,41 @@ Async APIs
     /**
      * Creates one or many models.
      * @param models The list of models to create. Each string corresponds to exactly one model.
-     * @return The list of created models
+     * @return A {@link PagedFlux} of created models and the http response.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public Mono<List<ModelData>> createModels(List<String> models) { }
-
-    /**
-     * Creates one or many models.
-     * @param models The list of models to create. Each string corresponds to exactly one model.
-     * @return A REST response containing the list of created models.
-     */
-    @ServiceMethod(returns = ReturnType.COLLECTION)
-    public Mono<Response<List<ModelData>>> createModelsWithResponse(List<String> models) { }
+    public PagedFlux<ModelData> createModels(List<String> models) { }
 
     /**
      * Gets a model, including the model metadata and the model definition.
      * @param modelId The Id of the model.
-     * @return The application/json model
+     * @return The ModelData
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<ModelData> getModel(String modelId) { }
 
     /**
      * Gets a model, including the model metadata and the model definition asynchronously.
      * @param modelId The Id of the model.
-     * @return A REST response containing the model.
+     * @return The ModelData and the http response
```

Comment:
I have seen the format of "The Http response containing modelData" 
We can decide one and use the same everywhere.